x-pack/plugins/cases/public/components/use_breadcrumbs/index.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { ChromeBreadcrumb } from 'kibana/public';
+import { useCallback, useEffect } from 'react';
+import { useKibana, useNavigation } from '../../common/lib/kibana';
+import { CasesDeepLinkId, ICasesDeepLinkId } from '../../common/navigation';
+import { useCasesContext } from '../cases_context/use_cases_context';
+
+const casesBreadcrumbTitle: Record<ICasesDeepLinkId, string> = {
+  [CasesDeepLinkId.cases]: i18n.translate('xpack.cases.breadcrumbs.all_cases', {
+    defaultMessage: 'Cases',
+  }),
+  [CasesDeepLinkId.casesCreate]: i18n.translate('xpack.cases.breadcrumbs.create_case', {
+    defaultMessage: 'Create',
+  }),
+  [CasesDeepLinkId.casesConfigure]: i18n.translate('xpack.cases.breadcrumbs.configure_cases', {
+    defaultMessage: 'Configure',
+  }),
+};
+
+function getTitleFromBreadCrumbs(breadcrumbs: ChromeBreadcrumb[]) {
```

Comment:
nit: I think it is easier to read the function signature if there is a return type (`string[]` in this case).